com.unity.shadergraph/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/MatrixTransposeNode.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections.Generic;
+using static UnityEditor.ShaderGraph.Registry.Types.GraphType;
+
+namespace com.unity.shadergraph.defs
+{
+
+    internal class MatrixTransposeNode : IStandardNode
+    {
+        public static FunctionDescriptor FunctionDescriptor => new(
+            1,
+            "MatrixTranspose",
+            "Out = transpose(In);",
+            new ParameterDescriptor("In", TYPE.Matrix, Usage.In),
+            new ParameterDescriptor("Out", TYPE.Matrix, Usage.Out)
+        );
+
+        public static Dictionary<string, string> UIStrings => new()
+        {
+            { "Name.Synonyms", "Matrix Transpose" },
```

Comment:
it's redundant to have a synonym that's the same as the node's name. I think it would work better if the synonym were just "transpose"